templates/cluster.yml.j2

```diff
@@ -4,6 +4,14 @@ metadata:
   creationTimestamp: null
   name: {{ cluster.name }}
 spec:
+  kubelet:
+    anonymousAuth: false
+    authenticationTokenWebhook: true
+    authorizationMode: Webhook
+  kubeControllerManager:
+    enableProfiling: false
+  kubeScheduler:
+    enableProfiling: false
```

Comment:
@titanlien I would variablize this with sane default values as set here already.

Reply:
Updated